Meshconvert/Meshconvert.cpp

```diff
@@ -110,11 +113,40 @@ const SValue g_pOptions[] =
     { L"flipu",     OPT_FLIPU },
     { L"flipv",     OPT_FLIPV },
     { L"flipz",     OPT_FLIPZ },
+    { L"fn",        OPT_VERT_NORMAL_FORMAT },
+    { L"fuv",       OPT_VERT_UV_FORMAT },
+    { L"fc",        OPT_VERT_COLOR_FORMAT },
     { L"nologo",    OPT_NOLOGO },
     { L"flist",     OPT_FILELIST },
     { nullptr,      0 }
 };
 
+const SValue g_vertexNormalFormats[] =
+{
+    { L"float3",    DXGI_FORMAT_R32G32B32_FLOAT },
```

Comment:
These names match up with D3D9 era decl names and the inputs to the Content Exporter options.